cccatalog-api/test/api_live_integration_test.py

```diff
@@ -130,3 +130,34 @@ def test_list_delete(test_list_create):
         verify=False
     )
     assert response.status_code == 204
+
+
+def test_license_type_filtering():
+    """
+    Ensure that multiple license type filters interact together correctly.
+    """
+    commercial = {'BY', 'BY-SA', 'BY-ND', 'CC0', 'PDM'}
```

Comment:
It would be nice to store lists of all licenses, commercial licenses, modification licenses, etc. in constants somewhere (maybe as attributes of the `Image` model so you could use it to create a `choices` list for the `license` field, or maybe Django settings) so that if they changed, you only had to change the list in one place. That might be out of scope for this PR, though.

Reply:
The license logic is hardcoded in `licenses.py`; I can change the tests to make use of those.

Reply:
(change made)